android/appv2/src/main/java/com/adammcneilly/pocketleague/android/app/MainActivity.kt

```diff
@@ -21,11 +28,26 @@ class MainActivity : ComponentActivity() {
 
         WindowCompat.setDecorFitsSystemWindows(window, false)
 
+        val database = PocketLeagueDB(DatabaseDriverFactory(this).createDriver())
+
+        val eventRepository = OfflineFirstEventRepository(
+            localEventService = SQLDelightEventService(
+                database = database,
+            ),
+            localMatchService = SQLDelightMatchService(
+                database = database,
+            ),
+            remoteEventService = OctaneGGEventService(),
+            remoteMatchService = OctaneGGMatchService(),
+        )
```

Comment:
We should pull this out of main activity. Or, at the very least, instead of supplying stuff directly we can create an app module like we did in old app version.